backend/btrixcloud/orgs.py

```diff
@@ -660,6 +673,71 @@ async def update_quotas(self, org: Organization, quotas: OrgQuotasIn) -> None:
                     {"$inc": {"giftedExecSecondsAvailable": gifted_secs_diff}},
                 )
 
+    async def add_to_org_quotas(
```

Comment:
Given the similarity between this and the update_quotas method, I wonder if it would be easier to maintain to just combine adding into the previous method, and use $inc vs $set, to avoid duplicating a lot of logic..

Reply:
Could still be a separate endpoint, but just one function to do both here.

Reply:
Sure! The quota history field uses the quota values after update, rather than the changes themselves, which means that in order to get those values we need to get the value of the quotas field after applying the `$inc` updates, so there are a couple places in the combined method where we need to do things in different orders (specifically, there's a call to `find_one_and_update` with `return_document=ReturnDocument.AFTER` to get the value of the quotas after the `$inc`). Done in cb43059.